state-sync/storage-service/server/src/lib.rs

```diff
@@ -121,7 +135,29 @@ impl<T: StorageReaderInterface> Handler<T> {
         Self { config, storage }
     }
 
-    pub fn call(&self, request: StorageServiceRequest) -> Result<StorageServiceResponse> {
+    pub fn call(
+        &self,
+        peer: PeerId,
+        protocol: ProtocolId,
+        request: StorageServiceRequest,
+    ) -> Result<StorageServiceResponse> {
+        // Update the request count
+        increment_counter(
+            &metrics::STORAGE_REQUESTS_RECEIVED,
+            peer,
+            protocol,
+            request.to_label().into(),
```

Comment:
Here's a perfect example, since you're `into()` in one direction and `into()` in the other direction

Reply:
I don't feel strongly. Changed to `get_label()`.